packages/manager/modules/pci/src/projects/project/instances/instance/detach-private-network/detach-private-network.component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './detach-private-network.controller';
+import template from './detach-private-network.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    projectId: '<',
+    instance: '<',
+    privateNetworks: '<',
```

Comment:
Is privateNetworks binding needed ? I don't see any reference for this.

Reply:
Yes, it is needed because we get the privateNetworks from the parent component (we don't do another API call) as it is the case for `attach-private-network.component.js`.